JASP-Desktop/gui/jaspversionchecker.cpp

```diff
@@ -47,6 +56,14 @@ void JASPVersionChecker::downloadFinished()
 	if (latest > cur)
 		emit showDownloadButton(downloadfile);
 
+	if(KnownIssues::issues()->downloadNeededOrLoad())	downloadKnownIssues();
```

Comment:
This is double: downloadKnownIssues is already called in downloadNeededOrLoad

Reply:
No it isn't.

Show me where then?